### PR TITLE
RI-4351 - add key names in recommendation

### PR DESCRIPTION
**Pull request by AmirAllayarovSofteq**

### CODE REVIEW

**Review by arthosofteq — APPROVED** [review]
looks good. you've migrated to repository - nice 
Please address comment about clientMetadata

File: redisinsight/api/src/modules/database-recommendation/repositories/local.database.recommendation.repository.spec.ts [hunk 1]
```diff
@@ -0,0 +1,251 @@
+// import { Test, TestingModule } from '@nestjs/testing';
```
[the remark this answers is not in the document]
Comment (AmirAllayarovSofteq) [reply redisinsight/api/src/modules/database-recommendation/repositories/local.database.recommendation.repository.spec.ts:1]: Yep, thank you, i'm working on it

File: redisinsight/api/src/modules/database-recommendation/database-recommendation.module.ts [hunk 2]
```diff
@@ -1,26 +1,38 @@
-import { Module } from '@nestjs/common';
+import { Module, Type } from '@nestjs/common';
 import { DatabaseRecommendationController }
   from 'src/modules/database-recommendation/database-recommendation.controller';
 import { DatabaseRecommendationService } from 'src/modules/database-recommendation/database-recommendation.service';
-import { DatabaseRecommendationProvider }
-  from 'src/modules/database-recommendation/providers/database-recommendation.provider';
+// import { DatabaseRecommendationProvider }
+//   from 'src/modules/database-recommendation/providers/database-recommendation.provider';
```
[the remark this answers is not in the document]
Comment (AmirAllayarovSofteq) [reply redisinsight/api/src/modules/database-recommendation/database-recommendation.module.ts:6]: Good catch, thank you

File: redisinsight/api/src/modules/database-recommendation/database-recommendation.controller.ts [hunk 3]
```diff
@@ -78,7 +80,7 @@ export class DatabaseRecommendationController {
       @BrowserClientMetadata() clientMetadata: ClientMetadata,
       @Body() dto: ModifyDatabaseRecommendationDto,
   ): Promise<DatabaseRecommendation> {
-    return await this.service.update(clientMetadata, id, dto);
+    return await this.service.update(id, dto);
```
[the remark this answers is not in the document]
Comment (AmirAllayarovSofteq) [reply redisinsight/api/src/modules/database-recommendation/database-recommendation.controller.ts:83]: We updating recommendation only by id, why we need clientMetadata here? (id it is enough)

**Pull request merged**